config/initializers/assets.rb

```diff
@@ -0,0 +1,14 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
leftover from rails 5 stuff?

Reply:
yes, it was  --- but if i remove it that removal gets merged into rails-5 branch.